apps/vs-code-designer/src/app/commands/dataMapper/DataMapperPanel.ts

```diff
@@ -428,15 +481,19 @@ export default class DataMapperPanel {
     });
   }
 
-  public getDataMapperVersion() {
-    const azureDataMapperConfig = workspace.getConfiguration(ext.prefix);
-    const configValue = azureDataMapperConfig.get<number>(dataMapperVersionSetting) ?? defaultDataMapperVersion;
+  public handleGetDataMapperVersion() {
     this.sendMsgToWebview({
       command: ExtensionCommand.getDataMapperVersion,
-      data: configValue,
+      data: this.dataMapVersion,
     });
   }
 
+  public getDataMapperVersion() {
```

Comment:
NIT: There is a `getGlobalSetting`, maybe reuse it? I know its will not make a huge difference so its ok if not